Source/WebCore/dom/Node.h

```diff
@@ -31,6 +31,7 @@
 #include "StyleValidity.h"
 #include "TaskSource.h"
 #include "TreeScope.h"
+#include "TrustedScript.h"
```

Comment:
We should try to avoid adding this header file; it will slow down the build to add another header to a commonly included header like `Node.h`. Instead we should see if we can use a forward declaration.

Reply:
That sounds oversimplified. We can compile `RefPtr<T>` with a forward declaration of T, but not necessarily _uses_ of `RefPtr<T>`. So we need to make sure the right `.cpp` files have the include but not include in the `.h` file. We have gotten this wrong before and slowed down the build a lot and then came back and fixed it.

Reply:
Possibly it's the combination of RefPtr and variant? It's definitely something I've tried with previous PRs and it not compile

Reply:
Okay this specifically does seem to work as a forward declaration, apologies I did try before but must have been doing something wrong or hitting a specific condition. I'll have a look elsewhere and do a follow up if needed for other places.